src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -31,6 +36,26 @@ public ConsoleLogRecordExporter(ConsoleExporterOptions options)
 
         public override ExportResult Export(in Batch<LogRecord> batch)
         {
+            if (this.disposed)
+            {
+                if (!this.isDisposeMessageSent)
+                {
+                    lock (this.disposeLockObj)
+                    {
+                        if (!this.isDisposeMessageSent)
+                        {
+                            this.isDisposeMessageSent = true;
+                            this.WriteLine("The console exporter is still being invoked after it has been disposed. This could be the result of invalid lifecycle management of the OpenTelemetry .NET SDK.");
```

Comment:
Do we really need these I/O operations to be inside the lock?

Reply:
updated, removed the lock